src/react-components/ui-root.js

```diff
@@ -335,10 +335,14 @@ class UIRoot extends Component {
 
     if (this.loadingTimeout) window.clearTimeout(this.loadingTimeout);
     this.loadingTimeout = window.setTimeout(() => {
-      if (!this.props.isBotMode) {
-        AFRAME.scenes[0].renderer.compileAndUploadMaterials(AFRAME.scenes[0].object3D, AFRAME.scenes[0].camera);
+      if (!this.state.hideLoader) {
```

Comment:
There's a race condition here since `setState` is async. I am bringing my branch up to speed where there's this same fix but flagging that you've been to this code path at least once is not done with react state.

Reply:
I'm fixing merge conflicts with that branch and HEAD at the moment. Will post in a bit.